### PR TITLE
changed the practice.css

### PR DESCRIPTION
**Pull request by priyanshu2k3**
Changed the practice.css file
 reduced the space under the div NameModal
so that there is no extra space left around

### CODE REVIEW

**Comment by arpitghura** [comment]
Hey, @priyanshu2k3 
Remember: Do mention about the issue that you resolved by this PR.

File: practice.css [hunk 1]
```diff
@@ -158,7 +158,7 @@ h2.subtitle {
 
 .nameModal{
   background-color: antiquewhite;
-  height: 40%;
+  height: 22.5%;
```
Comment (arpitghura) [inline practice.css:161]: You have to remove this property.
DONOT change the value as this creates unresponsiveness.
SEE: 
image

**Review by arpitghura — CHANGES_REQUESTED** [review]
Remove the height property.

File: practice.css (hunk 1 above)
Comment (priyanshu2k3) [reply practice.css:161]: ook trying to do that

**Comment by priyanshu2k3** [comment]
hey 
I have done as requested and also checked the responsiveness  its working fine

**Review by arpitghura — APPROVED** [review]

**Pull request merged**